test/fixtures/output/feedback/help-onprem.golden

```diff
@@ -0,0 +1,9 @@
+Submit feedback about the Confluent CLI.
```

Comment:
This command shouldn't exist on-prem!

Reply:
Did this file get generated incorrectly? (or maybe before you introduced the Cloud-only requirement)

Reply:
introduced before cloud-only, just deleted.